proto/backup.proto

```diff
@@ -124,6 +125,26 @@ message S3 {
     string secret_access_key = 9;
 }
 
+// GCS storage backend saves files into google cloud storage.
+message GCS {
```

Comment:
Should include `projectID`, `location`?

Reply:
We do not have to. These are for creating bucket.